tests/iio_common.c

```diff
@@ -126,6 +126,23 @@ struct iio_context * autodetect_context(bool rtn, const char * name, const char
 	return ctx;
 }
 
+ssize_t iio_device_enable_channel(const struct iio_device *dev, const char * channel, bool type)
+{
+	struct iio_channel *ch;
+	if (!channel)
+		return -EFAULT;
```

Comment:
I don't think you need to check that. This will never happen.

Reply:
today - but not sure about how others might use us :) 

Removed.